docs/_docs/configuration/menu-hjson.md

```diff
@@ -362,3 +363,188 @@ newUserApplicationPre: {
     // note that the rest of this menu is omitted for clarity
 }
 ```
+
+## Case Study: Manual Encoding Selection
+
+Enigma½ tries to automatically determine the proper encoding for a client when it connects. Unfortunately, there are cases where the wrong encoding can be selected, resulting in terminal programs that are not supported. If your user base contains users that would like to connect with unsupported clients, one solution is to offer manual encoding selection.
+
+A new system method has been added to support this, @systemMethod:setClientEncoding.
+
+### Simple example
+
+A basic config to use this could look something like the following:
+
+
+```hjson
+telnetConnected: {
+    art: CONNECT
+    next: clientSelectEncoding
+    config: { nextTimeout: 1500 }
+}
+
+clientSelectEncoding: {
+    art: CLTSEL.ASC
+    next: matrix
+    config: { interrupt: realtime }
+    form: {
+        0: {
+            mci: {
+                HM1: {
+                    submit: true
+                    hotKeys: { U: 0, C: 1 }
+                    hotKeysSubmit: true
+                    focus: true
+                    argName: encoding
+                    items: [
+                      {
+                          text: U) UTF-8
+                          data: utf-8
+                      }
+                      {
+                          text: C) CP437
+                          data: cp437
+                      }
+                    ]
+                }
+            }
+            submit: {
+              *: [
+                {
+                  value: { encoding: "utf-8" }
```

Comment:
If you wanted, you could just have a single handler and a value of null (match any)

Reply:
Hmm, so I need to play with that a bit more. I tried that, and it seems like it wouldn't handle at all in that case, though honestly I probably just had a silly mistake in the syntax or something, I'll revisit that thanks.

Reply:
Done. Ok I totally didn't know you could do that :) I went back and looked, and there is a comment in the documentation about this but I also added a bit more to hopefully make it obvious to other people as well, because that's a good feature.